src/Message/KPRefundOrderCheckResponse.php

```diff
@@ -29,8 +30,34 @@ public function getData(): array
     }
 
     public function isSuccessful()
+    {
+        return $this->response->getStatusCode() == 200;
+    }
+
+    public function isRefundAllowed()
+    {
+        $klarnaOrder = $this->responseBody;
+
+        $refundedAmount = 0;
```

Comment:
Is `$this->refundAmount` not accurate here? Seems like we are re-calculating it?